protocol/parser/parser.go

```diff
@@ -408,3 +424,14 @@ func appendInterfaceToInterfaceArray(value interface{}) []interface{} {
 	retArr = append(retArr, value)
 	return retArr
 }
+
+func convertInterfaceToString(input interface{}) (string, error) {
+	switch inp := input.(type) {
+	case string:
+		return inp, nil
+	case float64:
+		return fmt.Sprintf("%v", inp), nil
```

Comment:
there is a better way to convert float64 to string use strconv.FormatFloat() instead.

Reply:
Sure: 41f80bad125e2f5f36fc678183fe4c806426151e